gtsfm/multi_view_optimizer.py

```diff
@@ -77,9 +78,21 @@ def create_computation_graph(
 
         init_cameras_graph = dask.delayed(init_cameras)(wRi_graph, wti_graph, intrinsics_graph)
 
-        ba_input_graph = self.data_association_module.create_computation_graph(
+        data_assoc_graph = self.data_association_module.create_computation_graph(
             init_cameras_graph, v_corr_idxs_graph, keypoints_graph
         )
+        ba_input_graph = data_assoc_graph[0]
+        data_assoc_metrics_graph = data_assoc_graph[1]
```

Comment:
```
ba_input_graph, data_assoc_metrics_graph = self.data_association_module.create_computation_graph(
            init_cameras_graph, v_corr_idxs_graph, keypoints_graph
        )
```
Isn't that better?

Reply:
Dask won't allow this unfortunately:
```python
Traceback (most recent call last):
  File "gtsfm/runner/run_scene_optimizer_argoverse.py", line 97, in <module>
    run_scene_optimizer(args)
  File "gtsfm/runner/run_scene_optimizer_argoverse.py", line 37, in run_scene_optimizer
    sfm_result_graph = scene_optimizer.create_computation_graph(
  File "/Users/johnlambert/Documents/gtsfm/gtsfm/scene_optimizer.py", line 184, in create_computation_graph
    (ba_input_graph, ba_output_graph, optimizer_metrics_graph, ) = self.multiview_optimizer.create_computation_graph(
  File "/Users/johnlambert/Documents/gtsfm/gtsfm/multi_view_optimizer.py", line 81, in create_computation_graph
    ba_input_graph, data_assoc_metrics_graph = self.data_association_module.create_computation_graph(
  File "/Users/johnlambert/anaconda3/envs/gtsfm-v1/lib/python3.8/site-packages/dask/delayed.py", line 562, in __iter__
    raise TypeError("Delayed objects of unspecified length are not iterable")
TypeError: Delayed objects of unspecified length are not iterable
```

Reply:
This logic should be moved to `DataAssociation`'s `create_computation_graph`.

Reply:
I'm good with moving it, but is there a particular reason to put it in one place rather than the other?

Reply:
I think its more clear seeing the semantics on what to expect. If a module has three `Delayed` being returned, we can create richer docstrings.